src/holofuel/components/SideMenu/SideMenu.js

```diff
@@ -28,7 +28,7 @@ export function SideMenu ({
   return <aside styleName={cx('drawer', { 'drawer--open': isOpen })}>
     <div styleName='container'>
       <header styleName='header'>
-        <h1 styleName='appName'>HoloFuel</h1>
+        <h1 styleName='appName' data-testid='sidemenu-header'>HoloFuel</h1>
```

Comment:
Same as my comment in the other sidemenu. When there's static text we should use that to get the element rather than test-id

Reply:
Let's also discuss a rule of thumb here. :+1: :eyes: